test/Test/P/List.hs

```diff
@@ -19,6 +19,10 @@ prop_lastMaybe_empty :: Property
 prop_lastMaybe_empty =
   lastMaybe [] === (Nothing :: Maybe ())
 
+prop_count :: [Int] -> Property
+prop_count list =
+  count (>= 0) list === length (filter (>= 0) list) 
+
```

Comment:
With `Test.QuickCheck.Function` it is just
```haskell
prop_count :: [Int] -> (Fun Int Bool) -> Property
prop_count list (Fun _ f) =
  count f list === length (filter f list)
```
And it does produce something similar to
```
*** Failed! Falsifiable (after 1 test and 2 shrinks):  
[]
{_->False}
1 /= 0
```
Looks useful to me